src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -89,14 +110,21 @@
         <div class="hintbox" ng-show="!editorState.params.metric.colorsRange.length">
           <p>
             <i class="fa fa-danger text-danger"></i>
-            <strong>Required:</strong> You must specify at least one range.
+            <span
+              i18n-id="metricVis.params.ranges.warning.specifyRangeDescription"
+              i18n-default-message="{strongRequiredDescripion} You must specify at least one range."
+              i18n-values="{ strongRequiredDescripion: '<strong>' + editorState.requiredDescription + '</strong>' }"
```

Comment:
Why can't we just have the `i18n-default-message` (or any translate message) to fully contain that string, i.e.: `i18n-default-message="<strong>Required</strong> You must specify at least one range."`? cc @azasypkin

Reply:
@timroes I'm sorry, I was wrong about tags support in `defaultMessage` for angularJs directives. It's possible to apply `strong` tag for example and it will render correctly. But in react, core (`i18n.translate`) and angularjs filters it isn't allowed html in `defaultMessage`.
Nevertheless I think we should stick to common syntax for `defaultMessage` across react, angular and core.